articles/quickstart/spa/_includes/_user_profile_additional_info.md

```diff
@@ -1,3 +1,3 @@
-## Updating a User's Profile
+## Allow Users to Update Their Profile Information
 
-Auth0 provides a `user_metadata` object for all users. This object is intended to store information that can be modified by the user and is a useful place to keep data that is meant to be user-defined. See the [management API documentation](api/management/v2) for more information on how to allow users to make edits to their own profile information.
\ No newline at end of file
+You can allow your users to edit their profile information. Auth0 provides a `user_metadata` object to store information that users can edit. See the [management API documentation](api/management/v2) for more information.
```

Comment:
This guidance looks too generic. I'd link here https://auth0.com/docs/metadata (which links to this doc https://auth0.com/docs/metadata/management-api that has some examples on how to use the Mgmt API)